src/Cache/Exception/InvalidArgumentException.php

```diff
@@ -14,7 +14,7 @@
  * @since         3.7.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
-namespace Cake\Cache;
+namespace Cake\Cache\Exception;
```

Comment:
We'll need to add a class_alias for the old class. Userland code could have `catch` blocks that reference the old class name and we can't break those references.